Navigation/navigation.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Raleway&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="navigation.css">
+    <title>Navigation</title>
+</head>
+<body>
+    <div>
+        <ul class="navigation-container">
```

Comment:
Instead of using a list tag try using div with flexbox will look better and also in the future you can add branding or buttons and a custom layout for all, just a suggestion. 
This too is good.

Reply:
Sure, will do. Thanks for the suggestion!